src/client/component/workshop.cpp

```diff
@@ -72,6 +63,113 @@ namespace workshop
 		}
 	}
 
+	std::string get_mod_name(const std::string& mod_id)
+	{
+		if (mod_id == "usermaps")
+		{
+			return mod_id;
+		}
+
+		const utils::nt::library host{};
+		std::string path;
+
+		if (game::is_server())
+		{
+			const auto base_path = host.get_folder().generic_string();
+			path = utils::string::va("%s/mods/%s/zone/workshop.json", base_path.data(), mod_id.data());
+		}
+		else
+		{
+			return mod_id;
+		}
```

Comment:
why no early return here?

```suggestion
		if (!game::is_server())
		{
			return mod_id;
		}
		
		const utils::nt::library host{};
		const auto base_path = host.get_folder().generic_string();
		std::string path = utils::string::va("%s/mods/%s/zone/workshop.json", base_path.data(), mod_id.data());
```